### PR TITLE
Ewl 3183 article template

### PR DESCRIPTION
**Pull request by aCyborg**
The only functionality missing from this PR is the jump nav is-active switch. We couldn't get waypoint to work after extensively trying.

### CODE REVIEW

File: styleguide/source/assets/js/jump-nav.js [hunk 1]
```diff
@@ -9,4 +9,45 @@ $(document).ready(function() {
...
+        content.fadeIn(250);
+      }
+    };
+
+    $(window).on('scroll', onScroll);
+  };
+
+var demo = new StickyElement($('#sticky-jumpnav'));
+
 });
```
Comment (martensc) [inline styleguide/source/assets/js/jump-nav.js]: @aCyborg it looks like there are some js errors happening in the `jump-nav` and `featured-content` files. You'll need to add in the `jQuery noConflict` wrapper around each. Is this something you feel comfortable updating or should I take care of this?

File: styleguide/source/_patterns/03-templates/02-article/article.twig [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  <div class="layout_two_up-heavy-right-offset-secondary">
+    {# Share this molecule goes here #}
+  </div>
+  <div class="layout_two_up-heavy-right-offset-primary">
+    {% include 'molecules-list-rule' %}
+    {% include 'molecules-list-rule-metadata' %}
+  </div>
+</div>
+<div class="layout layout-one-up">
+  {% include 'organisms-related-content' %}
```
Comment (martensc) [inline styleguide/source/_patterns/03-templates/02-article/article.twig]: At about `934px` viewport the `.date-block_date` component type gets cutoff.
screen shot 2017-03-06 at 10 22 17 am

File: styleguide/source/assets/js/jump-nav.js (hunk 1 above)
Comment (aCyborg) [inline styleguide/source/assets/js/jump-nav.js]: If you could that would be great...or else point me in the direction of one that is working and I can try and hobble it together!

File: styleguide/source/_patterns/03-templates/02-article/article.twig (hunk 2 above)
Comment (aCyborg) [inline styleguide/source/_patterns/03-templates/02-article/article.twig]: I can take a stab at this. The design specifies a ratio, but maybe I can turn the ratio off at this breakpoint, or just reduce some spacing here so it fits but maintains the ratio. I'll take a look.

File: styleguide/source/assets/js/jump-nav.js (hunk 1 above)
Comment (martensc) [inline styleguide/source/assets/js/jump-nav.js]: @aCyborg done. It's pretty simple. Essentially you just need to modify the beginning and end of the jquery file.

At the beginning:
```
jQuery.noConflict();
(function($) {
```

At the end
```
})(jQuery);
```
Reference: https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/65/commits/612d02d1cf309e923ce5db42baf415aa7b315bc9

File: styleguide/source/assets/js/jump-nav.js [hunk 3]
```diff
@@ -9,4 +10,44 @@ $(document).ready(function() {
...
+        content.hide();
+      } else {
+        content.fadeIn(250);
+      }
+    };
+
+    $(window).on('scroll', onScroll);
+  };
+
+var demo = new StickyElement($('#sticky-jumpnav'));
```
Comment (martensc) [inline styleguide/source/assets/js/jump-nav.js]: @aCyborg does this variable need to be here? I don't believe `demo` is being used.

Comment (aCyborg) [inline styleguide/source/assets/js/jump-nav.js]: @martensc ... I'm not sure. I think it doesn't work if you don't have it. but we could change the var to something else if demo doesn't make sense

Comment (martensc) [inline styleguide/source/assets/js/jump-nav.js]: @aCyborg yup, you're right. :) Maybe rename `demo` to something more specific. Like `stickyNav` or something.

Also, the `is-active` class doesn't change depending on which section you're in.

Comment (aCyborg) [inline styleguide/source/assets/js/jump-nav.js]: @martensc 

yah I left a note about the is-active bit in the description. We (lauren and I) couldn't get it to work with waypoints and that was our best bet. The live site uses some data urls to do it. I think that will need to get resolved in a future ticket.

**Pull request merged**